CruCentralCoast/Database/DatabaseManager.swift

```diff
@@ -31,7 +31,21 @@ class DatabaseManager {
     // Objects listening to changes in the database
     private var listeners: [WeakRef<DatabaseListener>] = []
     
-    private init() {}
+    private init() {
+        // Handle database migrations
+        Realm.Configuration.defaultConfiguration = Realm.Configuration(
+            // Set the new schema version. This must be greater than the previously used
+            // version (if you've never set a schema version before, the version is 0).
+            schemaVersion: 1,
+            
+            // Set the block which will be called automatically when opening a Realm with
+            // a schema version lower than the one set above
+            migrationBlock: { migration, oldSchemaVersion in
+                if (oldSchemaVersion < 1) {
+                    // Added Person.movements
```

Comment:
did you mean to have something here?

Reply:
Nope, that is purely for documentation purposes. I added the variable "movements" on the Person class, but it didn't require any special handling right here. Sometimes these changes will require special changes. It was required for me to migrate the database though since I changed the data model.